pages/index.tsx

```diff
@@ -227,35 +223,11 @@ const Home: FC<IHomeProps> = ({ latestPosts, network, networkSocialsData }) => {
 	const { id: userId } = useUserDetailsSelector();
 	const router = useRouter();
 	const dispatch = useDispatch();
-	const [isIdentityUnverified, setIsIdentityUnverified] = useState<boolean>(false);
 	const [openContinuingModal, setOpenContinuingModal] = useState<boolean>(Boolean(router.query.identityVerification) || false);
 	const { resolvedTheme: theme } = useTheme();
 
 	useEffect(() => {
 		dispatch(setNetwork(network));
-		if (!api || !apiReady) return;
-		let unsubscribe: () => void;
-		const address = localStorage.getItem('identityAddress');
-		const identityForm = localStorage.getItem('identityForm');
-		const encoded_addr = address ? getEncodedAddress(address, network) : '';
-		if (!identityForm || !JSON.parse(identityForm)?.setIdentity) return;
-
-		api.derive.accounts
-			.info(encoded_addr, (info: DeriveAccountInfo) => {
-				const infoCall = info.identity?.judgements.filter(([, judgement]): boolean => judgement.isFeePaid);
-				const judgementProvided = infoCall?.some(([, judgement]): boolean => judgement.isFeePaid);
-
-				setIsIdentityUnverified(judgementProvided || !info?.identity?.judgements?.length);
-				if (!(judgementProvided || !info?.identity?.judgements?.length)) {
-					localStorage.removeItem('identityForm');
-				}
-			})
-			.then((unsub) => {
-				unsubscribe = unsub;
-			})
-			.catch((e) => console.error(e));
-
-		return () => unsubscribe && unsubscribe();
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [network, api, apiReady, userId]);
```

Comment:
we're setting network every time api or user changes.
This does not make sense.